### PR TITLE
view templates

### PR DESCRIPTION
**Pull request by vikispeaks**

### CODE REVIEW

File: ejs-templating/view-templates/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+<link rel="stylesheet" href="/css/styles.css" />
+```
+
+To load the files created in the **public** folder, add the below code to the `index.js` file.
+
+```js
+app.use(express.static(path.join(__dirname, "public")));
+```
+
+If we open `localhost:3000` now we can see that the styling we created is applied to the page.
```
Comment (avishekjana) [inline ejs-templating/view-templates/README.md:34]: If we open `localhost:3000`, now we can see that the styling we created is applied to the page.

File: ejs-templating/view-templates/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Next to add JavaScript files, let us create a new **js** folder inside **public** folder and add `script.js` file inside it.
```
Comment (avishekjana) [inline ejs-templating/view-templates/README.md:36]: Next, to add JavaScript files, let us create a new **js** folder inside **public** folder and add `script.js` file inside it.

File: ejs-templating/view-templates/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Let us, for now, add a simple `console.log` statement to check if the code is working.
+
+```js
+// script.js
+console.log("First JS import on an EJS application!");
+```
+
+Saving this file does not restart the nodemon as we have not configured it to listen to changes made in js files.
```
Comment (avishekjana) [inline ejs-templating/view-templates/README.md:45]: Saving this file does not restart the nodemon, as we have not configured it to listen to changes made in **js** files.

File: ejs-templating/view-templates/README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+Let us change that by updating the config of `package.json` scripts. Replace the `start` script with the below code.
```
Comment (avishekjana) [inline ejs-templating/view-templates/README.md:47]: Let us change that, by updating the config of `package.json` scripts. Replace the `start` script with the below code.

File: ejs-templating/view-templates/README.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+This makes sure that the nodemon listens for changes in all files with extensions `.js` and `.ejs`.
+
+We can restart nodemon using `npm start` and we can see that the changes are reflected on the browser when we open `localhost:3000`.
+
+## Organizing files using View Templates
+
+EJS allows us to split a single file into separate files, so we can have common files that we can reuse across the application.
+
+In our case, let us create a Header for the Todo application with the text "This is my Todo Application" as a common view template.
```
Comment (avishekjana) [inline ejs-templating/view-templates/README.md:61]: In our case, let us create a **Header** for the Todo application with the text "This is my Todo Application" as a common view template.

File: ejs-templating/view-templates/README.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+Let us create a new file called `header.ejs` in our application within the views folder.
```
Comment (avishekjana) [inline ejs-templating/view-templates/README.md:63]: Let us create a new file called `header.ejs` in our application within the `views` folder.

**Review by avishekjana — CHANGES_REQUESTED** [review]

File: ejs-templating/view-templates/README.md (hunk 1 above)
Comment (vikispeaks) [reply ejs-templating/view-templates/README.md:34]: Keeping this as it is. I think with respect to adding comma's the Language tool and Grammarly treat the sentence differently.

File: ejs-templating/view-templates/README.md (hunk 2 above)
Comment (vikispeaks) [reply ejs-templating/view-templates/README.md:36]: Same as above.

File: ejs-templating/view-templates/README.md (hunk 3 above)
Comment (vikispeaks) [reply ejs-templating/view-templates/README.md:45]: Done.

File: ejs-templating/view-templates/README.md (hunk 4 above)
Comment (vikispeaks) [reply ejs-templating/view-templates/README.md:47]: Same as above.

File: ejs-templating/view-templates/README.md (hunk 5 above)
Comment (vikispeaks) [reply ejs-templating/view-templates/README.md:61]: It is the same as stated review.

File: ejs-templating/view-templates/README.md (hunk 6 above)
Comment (vikispeaks) [reply ejs-templating/view-templates/README.md:63]: Done.

**Pull request merged**